astroquery/gaia/tests/test_gaiatap.py

```diff
@@ -16,6 +16,8 @@
 """
 import unittest
 import os
+import pytest
```

Comment:
(Also while I was doing that PR, came to realize that it doesn't matter where to import in the individual test files it will still use astropy unless ``ASTROPY_USE_SYSTEM_PYTEST`` is set).